.github/workflows/create-cherry-pick-pr.yml

```diff
@@ -0,0 +1,18 @@
+name: Create cherry-pick PR
+on:
+  pull_request_target:
+    branches:
+      - 'v*.x'
+      - 'master'
+    types: ['closed']
+
+permissions: {}
+
+jobs:
+  create_pr:
+    name: Create cherry-pick PR
+    uses: mui/mui-public/.github/workflows/prs_create-cherry-pick-pr.yml@master
+    permissions:
+      contents: write
+      issues: write
```

Comment:
This looks suspicious: https://github.com/mui/mui-public/pull/245

```suggestion
```

#15456 opened